lua/harpoon/utils.lua

```diff
@@ -1,8 +1,14 @@
+local Path = require("plenary.path")
 local M = {}
 
+function M.normalize_path(buf_name, root)
+    return Path:new(buf_name):make_relative(root)
+end
+
 function M.trim(str)
-   return str:gsub("^%s+", ""):gsub("%s+$", "")
+    return str:gsub("^%s+", ""):gsub("%s+$", "")
```

Comment:
lgtm! Except for this singular extra space here lol

Reply:
4 space indenting, thats where its meant to be

Reply:
LOL idk how I didn't see that, my bad.